nss-day-cohort-29/welcome-to-nashville-daggers
# added folder structure and GRUNT!

# Description

Uploaded folder structure and Grunt functionality.


## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made

Fetch the branch and do two things: 
1. Check that Grunt runs from the lib directory.
2. Check that node_modules is grayed out in Visual Studio Code--this indicates that .gitignore is working and won't upload the node_modules to Github.


# Checklist:
